tests/e2e/e2e.go

```diff
@@ -199,3 +209,28 @@ func getE2EBaseURIs(tc tests.TestContext) []string {
 func formatURI(baseURI string, blockchainID ids.ID) string {
 	return fmt.Sprintf("%s/ext/bc/%s", baseURI, blockchainID)
 }
+
+func SetupDefaultChainAlias(chainID ids.ID, tc tests.TestContext) {
+	require := require.New(ginkgo.GinkgoT())
+
+	baseURI := "http://localhost:9650"
+	adminClient := admin.NewClient(baseURI)
+
+	aliases, err := adminClient.GetChainAliases(tc.DefaultContext(), chainID.String())
+	require.NoError(err)
+
+	fmt.Println("DEBUG: aliases", aliases)
+	fmt.Println("DEBUG: chainID", chainID)
```

Comment:
Can we remove these?

I think we can just remove these, but if we're going to add prints here, we should follow the style from the rest of the file of using `utils.Outf(...)`